build.sbt

```diff
@@ -0,0 +1,55 @@
+inThisBuild(Seq(
+  organization := "org.portable-scala",
+  version := "1.0.0-SNAPSHOT",
+
+  scalaVersion := "2.12.3",
+  scalacOptions ++= Seq("-deprecation", "-feature", "-encoding", "UTF-8"),
+
+  crossSbtVersions := Seq("1.0.2", "0.13.16"),
```

Comment:
add: `scalaCrossVersions := Seq("2.10.6", "2.12.3")`?

Reply:
That would be misleading. It would suggest that one can do `+test`. `crossSbtVersions` is meaningful because it's possible to do `^scripted`, for example.